tests/baselines/reference/checkJsxChildrenProperty13.errors.txt

```diff
@@ -14,8 +14,8 @@ tests/cases/conformance/jsx/file.tsx(12,30): error TS2710: 'children' are specif
         render() {
             // Error children are specified twice
             return (<InnerButton {...this.props} children="hi">
-                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-!!! error TS2710: 'children' are specified twice. The attribute named 'children' will be overwritten.
+                                                 ~~~~~~~~~~~~~
+!!! error TS2735: 'children' is overwritten by a later spread.

```

Comment:
...later? Methinks this needs a different error message, since it's not being overridden by a spread, but by the actual children tags in the AST.